src/bin/bat/main.rs

```diff
@@ -227,9 +227,24 @@ pub fn list_themes(cfg: &Config, config_dir: &Path, cache_dir: &Path) -> Result<
     Ok(())
 }
 
+fn set_terminal_title_to_inputs_names(inputs: &Vec<Input>) {
+    let mut input_names = "bat: ".to_string();
+    for (index, input) in inputs.iter().enumerate() {
+        input_names += &input.description.name.to_string();
+        if index < inputs.len() - 1 {
+            input_names += ", ";
+        }
+    }
+    print!("\x1b]2;{}\x07", input_names);
```

Comment:
Will this behave well in all major terminal emulators on all major OSes?

Reply:
I believe ANSI escape characters are standard across terminal emulators on different OSes, https://en.wikipedia.org/wiki/ANSI_escape_code#OSC_(Operating_System_Command)_sequences

I have updated the `2` in this line to `0` to match those docs